cfme/tests/containers/test_overview_data_integrity.py

```diff
@@ -47,9 +47,15 @@ def get_api_object_counts(appliance):
             out[Project] += len(provider.mgmt.list_project())
             out[Route] += len(provider.mgmt.list_route())
             # TODO image count? list_templates too high, list_image_stream_images too low
-            # TODO container count? list_container returning different count than MIQ reports
             out[ImageRegistry] += len(provider.mgmt.list_image_registry())
-            out[Container] += len(provider.mgmt.list_container())
+            listed_containers = provider.mgmt.list_container()
+            # Get all container pods
+            count_containers = []
+            for containers in listed_containers:
+                for container in containers:
+                    count_containers.append(container.name)
```

Comment:
We don't actually need the names, we really just want the count of inner items.

I'd suggest using something like:
```python
count_containers = sum(1 for pod in list_containers for c in pod)
```